.github/workflows/release-publish.yml

```diff
@@ -121,15 +121,15 @@ jobs:
           BUILD_ARGS="${BUILD_ARGS} -Dmaven.deploy.skip -Dgpg.skip"
         fi
 
-        ./mvnw -B ${GOALS} --file pom.xml ${PROFILES} ${NEXUS_ARGS} ${BUILD_ARGS}
+        ./mvnw --batch-mode --threads 1C ${GOALS} ${PROFILES} ${NEXUS_ARGS} ${BUILD_ARGS}
```

Comment:
Is the `nexus-staging-maven-plugin` thread safe?
I know it's _marked_ as thread-safe, but I'm not sure TBH and would prefer not to risk anything for the Nessie release process.

Reply:
i think being marked as thread-safe is the strongest indicator of supporting it correctly there is... but we can remove the parameter here just for the release workflow